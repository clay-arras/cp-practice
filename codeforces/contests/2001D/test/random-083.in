1
25
6 4 15 8 10 24 10 10 14 25 13 16 8 16 7 16 15 12 13 18 19 18 9 15 21
