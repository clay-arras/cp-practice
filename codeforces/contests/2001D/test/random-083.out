16
6 4 24 10 14 25 16 8 7 12 13 18 19 9 15 21
